### PR TITLE
Add Membrane.Time.to_timebase/2 function

### PR DESCRIPTION
**Pull request by varsill**

### CODE REVIEW

File: lib/membrane/time.ex [hunk 1]
```diff
@@ -226,6 +226,14 @@ defmodule Membrane.Time do
     round(value / native_unit())
   end
 
+  @doc """
+  Returns timestamp in timebase units. Rounded to the nearest integer.
+  """
+  @spec to_timebase(number | Ratio.t(), number | Ratio.t()) :: integer
+  def to_timebase(timestamp, timebase) do
```
Comment (mat-hek) [inline lib/membrane/time.ex:233]:
```suggestion
  @spec round_to_timebase(t, t | Ratio.t()) :: integer
  def round_to_timebase(timestamp, timebase) do
```

File: lib/membrane/time.ex [hunk 2]
```diff
@@ -226,6 +226,14 @@ defmodule Membrane.Time do
     round(value / native_unit())
   end
 
+  @doc """
+  Returns timestamp in timebase units. Rounded to the nearest integer.
```
Comment (mat-hek) [inline lib/membrane/time.ex:230]: an example would be useful

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/membrane/time.ex (hunk 1 above)
Comment (varsill) [reply lib/membrane/time.ex:233]: done

File: lib/membrane/time.ex (hunk 2 above)
Comment (varsill) [reply lib/membrane/time.ex:230]: done

File: lib/membrane/time.ex [hunk 3]
```diff
@@ -228,9 +228,15 @@ defmodule Membrane.Time do
 
   @doc """
   Returns timestamp in timebase units. Rounded to the nearest integer.
+
+  ## Examples:
+      iex> timestamp = 10 |> Membrane.Time.seconds()
+      iex> timebase = Ratio.new(Membrane.Time.second(), 30)
+      iex> Membrane.Time.round_to_timebase(timestamp, timebase)
+      300
```
Comment (mat-hek) [inline lib/membrane/time.ex:236]: you can make this a doctest

**Review by mat-hek — APPROVED** [review]

**Pull request merged**